config/locales/en.yml

```
@@ -54,6 +54,12 @@ en:
       what_wrong: "What went wrong?"
       send_me_survey: "Send me the survey"
       send: "Send"
+    input:
+      show: Show
+      hide: Hide
+      password: password
```

I like that the translations have been thought about - i18n ftw. 

I don't think that splitting it out into words is best practise - I think the translation would be better as phrases for 'show password' and 'hide password'. In different languages the word 'show' (meaning appear) might need the context of 'password' to be accurate (and visa versa) - and splitting them up individual words doesn't allow that.

```suggestion
      show: Show password
      hide: Hide password
```

---

The reason I split them is because 'password' gets wrapped in a span that gets hidden when the screen is narrower (on mobile but not below, I think) and I didn't want to put markup in the translation file.

---

Hmm, tricky. Hiding the word 'password' then runs into the same translation issue of potentially removing the context for 'Show'. This is why i18n is difficult.

Is there a layout where the word 'password' doesn't need to be hidden?

---

That would have to be a design question, I think. Unfortunately the phrase is quite long and takes up a fair bit of width on the smallest mobiles. Unless we drop it down below the input on mobile, as we do below 320px. I can see if that would be an option.

---

Done, see latest commit.